sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/command_component.py

```diff
@@ -50,11 +61,18 @@ class Meta:
     )
     properties = fields.Dict(keys=fields.Str(), values=fields.Raw())
 
+    additional_includes = fields.List(UnionField([fields.Str(), NestedField(AzureDevopsArtifactsSchema)]))
```

Comment:
Is this a public feature? If yes, maybe we can mention in changelog. Otherwise, shall we guard this under private preview flag.

Reply:
Not sure, @wangchao1230 for update

Reply:
added